lib/offline/download_manager.js

```diff
@@ -108,6 +120,14 @@ shaka.offline.DownloadManager = class {
             shaka.util.Error.Code.OPERATION_ABORTED);
       }
 
+      // Update initData
+      if (initSegment) {
+        const segmentBytes = shaka.util.BufferUtils.toUint8(response);
+        if (shaka.util.Pssh.containsPssh(segmentBytes, true)) {
+          this.onInitData_(segmentBytes);
```

Comment:
I wonder if it might be better to pass only the PSSH bytes instead of the whole segment.  I'm not certain what every browser will do with this, since the whole segment might not be understood as init data.